Added Microsoft Edge browser support

marcoaam opened the pull request.

adiel commented on src/Coypu.AcceptanceTests/ApiExamples.cs:
```diff
@@ -646,6 +646,7 @@ private static RemoteWebDriver CustomProfile()
         [TestCase("Windows 7", "firefox", "25")]
         [TestCase("Windows XP", "internet explorer", "6")]
+        [TestCase("Windows 10", "MicrosoftEdge", "14")]
         public void CustomBrowserSession(string platform, string browserName, string version)
```
HI thanks for this! Did you actually run this test? I would have thought you'd get `BrowserNotSupported` unless you also added some code to construct the correct selenium driver here:

https://github.com/featurist/coypu/blob/c86ac64d3f82aeb184eeeca1ee10b6b9850e86f1/src/Coypu/Drivers/Selenium/DriverFactory.cs

marcoaam replied:
Hi, Yes I did. The way the test is implemented it doesn't require changes to the DriverFactory.cs as in this case it is passing in the RemoteWebDriver in the constructor so it is not making use of the DriverFactory.cs, which is the same case that I have in my codebase. There is no harm in adding it though. I can submit another pull request with the changes and remove the commits of the bump of the version and the history.md changes if you prefer.

adiel commented on Coypu.nuspec:
```diff
@@ -2,7 +2,7 @@
   <metadata>
     <id>Coypu</id>
-    <version>2.11.0</version>
+    <version>2.12.0</version>
```
Usually a one or more changes are combined into a package release by whoever is maintaining the Nuget package. As such it's better to leave this alone and have just the code changes in this PR

adiel commented on History.md:
```diff
@@ -1,3 +1,11 @@
+
+Release date: 2016-12-22
+
 # Version 2.11.0
```
Usually a one or more changes are combined into a package release by whoever is maintaining the Nuget package. As such it's better to leave this alone and have just the code changes in this PR

marcoaam commented:
I have added support for MicrosoftEdge now in the DriverFactory.cs as well and added a couple of tests to cover it. You will need IE driver and MicrosoftWebDriver in your bin folder to be able to run these new tests.

Jetski5822 commented:
Thanks
